src/components/victory-scatter/point.jsx

```diff
@@ -11,7 +11,7 @@ export default class Point extends React.Component {
       x: React.PropTypes.any,
       y: React.PropTypes.any
     }),
-    labelComponent: React.PropTypes.element,
+    label: React.PropTypes.element,
```

Comment:
Should this be `labels` everywhere. I guess it's possible that someone might want to pass in an array of labels for each scatter. Either way, I'd like to be consistent with the other chart types.